src/firebolt/async_db/cursor.py

```diff
@@ -219,6 +223,13 @@ async def execute(
         set_parameters: Optional[Dict] = None,
     ) -> int:
         """Prepare and execute a database query. Return row count."""
+
+        # our CREATE EXTERNAL TABLE queries currently require credentials,
+        # so we will skip logging those queries.
+        # https://docs.firebolt.io/sql-reference/commands/ddl-commands#create-external-table
+        if not re.search("aws_key_id|credentials", query, flags=re.IGNORECASE):
+            logger.info(f"Running query: {query}")
```

Comment:
Maybe outside of scope of this change, but later on we could log those queries as well, replacing credentials with ***. It shouldn't be a massive effort since SQL is fairly structured and those clauses will be the same unless malformed.